seqr/views/apis/saved_variant_api.py

```diff
@@ -237,7 +219,7 @@ def update_variant_functional_data_handler(request, variant_guids):
         get_tag_create_data=lambda tag, **kwargs: {'functional_data_tag': tag.get('name')})
 
 
-def _update_variant_tag_models(request, variant_guids, tag_key, model_cls, get_tag_create_data, response_guid_key=None, delete_variants_if_empty=False):
+def _update_variant_tag_models(request, variant_guids, tag_key, model_cls, get_tag_create_data, response_guid_key=None, delete_variants_if_empty=False, protected_tag_type=None):
```

Comment:
why is AIP a protected tag? what does protected mean here?

Reply:
Per the ticket:
> Create a new AIP tag with a hover over for metadata, which include the AIP categories and date. This tag should not be editable by users

The fact that it should not be editable by users, and can only be added by the AIP upload and never removed by a user, is what "protected" means

Reply:
ok just making sure my understanding is correct: 

Since it's a protected tag type, the link between a saved variant and an AIP tag will not be editable by a user. The tag is associated with a saved variant (aka variant tag type model is created) when the AIP data is loaded.